pkg/openstack/test.go

```diff
@@ -0,0 +1,72 @@
+package openstack
+
+import (
+	"context"
+
+	"github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/configmap"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/helper"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/util"
+
+	corev1beta1 "github.com/openstack-k8s-operators/openstack-operator/apis/core/v1beta1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const (
+	// config map generated by Keyston operator containing clouds.yaml
+	CloudsConfigMap       = "openstack-config"
+	TestOperatorConfigMap = "test-operator-config"
+
+	TempestImageKey       = "tempest-image"
+	TobikoImageKey        = "tobiko-image"
+	HorizontestImageKey   = "horizontest-image"
+	AnsibletestImageKey   = "ansibletest-image"
+	CloudsKey             = "clouds-secret-name"
+)
+
+func getString(s *string) string {
```

Comment:
Could you use getImg for https://github.com/openstack-k8s-operators/openstack-operator/blob/main/pkg/openstack/version.go#L64

Reply:
Let me know please what you think about my update. The reason why we need to check whether the variable contains `nil` is that in downstream it might happen that certain test image does not have a build (RELATED_IMAGE_ env var is not set). But maybe I'm missing something.